.github/workflows/cd_release.yml

```diff
@@ -369,6 +369,7 @@ jobs:
     - name: Build (& deploy MkDocs) documentation
       run: |
         REF=${{ github.ref }}
```

Comment:
should it be put in one line or with different variable names for readability?

Reply:
Doing it in this way (using `${{ github.ref }}` it is not possible to do it in a one-liner).
For this, I don't see a reason to rename, but if, then it should be the 2nd variable name that should be changed.

Another solution is to use the equivalent environment variable that's set: `$GITHUB_REF`. With this it could be a one-liner.

Reply:
I'll implement the latter.

Also the other place where I'm doing the same thing (not shown in this PR at the moment).